lib/modules/datasource/index.ts

```diff
@@ -506,12 +505,6 @@ export async function getPkgReleases(
   return res;
 }
 
-export function getPkgReleasesSafe(
```

Comment:
looks like like unrelated change?

Reply:
It's related to the place when this function was called. When I simplified it's calling code, turned out this function isn't needed anymore. And I needed to change calling code at the `lookup` function anyway. I think nothing will break after "inlining" of this function.

Reply:
@rarkins added this function to leave the API compatible for our own tools. this will now break and I need to fix it after merge 😏

Reply:
Is it ok for you?

Reply:
sure, just was a little confused because the PR title and description told me something different than removing this function.